pyblish_nuke/lib.py

```diff
@@ -278,15 +279,15 @@ def dock(window):
 
         dialog.deleteLater()
 
-        # if there are more than one main window, its a floating window
+        # If there are more than one main window, its a floating window
         if len(main_windows) > 1:
```

Comment:
Don't worry about it, there's no right or wrong here. Just an opportunity to bring it up.